tasty/src/dotty/tools/tasty/TastyHeaderUnpickler.scala

```diff
@@ -107,11 +107,20 @@ object TastyHeaderUnpickler {
     }
     val minorVersion = showMinorVersion(MinorVersion, ExperimentalVersion)
     val fileMinorVersion = showMinorVersion(fileMinor, fileExperimental)
-    s"""TASTy signature has wrong version.
-      | expected: {majorVersion: $MajorVersion, minorVersion: $minorVersion}
-      | found   : {majorVersion: $fileMajor, minorVersion: $fileMinorVersion}
-      |
-      |""".stripMargin
+
+    if (MajorVersion == 28 && fileMajor == 28)
+      s"""Found a file on classpath, that was compiled with a newer Scala version than the one currently used.
```

Comment:
```suggestion
      s"""Found a file on classpath that was compiled with a newer Scala version than the one currently used.
```

No comma needed here.